sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java

```diff
@@ -64,4 +69,14 @@ private static boolean shouldServiceVersionBeTested(ConfigurationServiceVersion
         return Arrays.stream(configuredServiceVersionList).anyMatch(configuredServiceVersion ->
             serviceVersion.getVersion().equals(configuredServiceVersion.trim()));
     }
+
+    static TokenCredential getTokenCredential(InterceptorManager interceptorManager) {
+        if (interceptorManager.isPlaybackMode()) {
+            return new MockTokenCredential();
+        } else if (interceptorManager.isRecordMode()) {
+            return new DefaultAzureCredentialBuilder().build();
+        } else {
+            return new AzurePowerShellCredentialBuilder().build();
```

Comment:
This won't work in our current beta; all the params moved to required params on the builder. Ex:
```
  AzurePipelinesCredential credential = new AzurePipelinesCredentialBuilder()
      .clientId(clientId)
      .tenantId(tenantId)
      .serviceConnectionId(serviceConnectionId)
      .systemAccessToken(systemAccessToken)
      .build();
 ```

Reply:
This is using Powershell Credential here, not PipelinesCredential.

Reply:
PipelinesCredential support will be added later on as we update our Pipelines config.